pilot/cmd/pilot-agent/main.go

```diff
@@ -86,7 +86,7 @@ var (
 			}
 
 			// set values from registry platform
-			if role.IPAddress == "" {
+			if len(role.IPAddress) == 0 {
```

Comment:
Curious - why did you change this ? Is it considered more idiomatic ? Linter happier ?